lib/models/instance.js

```diff
@@ -139,6 +138,7 @@ Instance.prototype.listenToSocketEvents = function () {
  * @return {[type]}      [description]
  */
 Instance.prototype.handleSocketData = function (data) {
+  /*jshint maxcomplexity:12 */
```

Comment:
In relation to changing to `semistandard`. 

1. Seems like it'd be a bit of work

```
$ semistandard lib/**/* | grep external -v | wc -l
semistandard: Semicolons For All! (https://github.com/Flet/semistandard)
167
```

2. I'd rather just full out switch to `standard` if we're going to switch since that's what we're using in most places. The browser doesn't care about semicolons either.